quickwit/rest-api-tests/scenarii/es_compatibility/0005-query_string_query.yaml

```diff
@@ -151,3 +151,28 @@ json:
       lenient: true
       query: "to AND the"
 status_code: 400
+---
+# trailing wildcard
+json:
+  query:
+    query_string:
+      default_field: payload.description
+      lenient: true
+      query: "Jour* AND unix"
+expected:
+  hits:
+    total:
+      value: 2
+---
+# trailing wildcard
+json:
```

Comment:
Is it possible to add a test that returns more than 50 hits so that we can see the max_expansions limit?